package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
```

Comment:
@ashusharma-git The `package.json` should have more madatory fields such as name, description, etc. Then name can be `exponential-utils`. remove this package.json from your local directory, and generate one using command `npm init`. It will help adding mandatory fields